scipy/integrate/_py/base.py

```diff
@@ -61,12 +61,15 @@ class OdeSolver(object):
         Current time.
     y : ndarray
         Current state.
+    t_old : float
+        Time of the last successful step. None if no steps were made yet.
```

Comment:
Really minor, but this descriptions is not very clear in my opinion. Maybe "End time of the last successful step" (the main complaint is that "time" is too vague).